code/src/components/ThirdQuestion.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export const ThirdQuestion = ({text, setText}) => {
+    return (
+        <div className='third-question'>
+            <label><h2 className='text-header' tabIndex='0'>What was the main reason you wanted to start at the Technigo bootcamp and become a frontend developer?</h2></label>
+                <textarea
+                    aria-label='text area'
```

Comment:
Its great that you used the text area and limiting the number of characters. It really looks very good on the site.